jurity/recommenders/base.py

```diff
@@ -103,6 +107,23 @@ def get_score(self, actual_results: pd.DataFrame, predicted_results: pd.DataFram
         """
         pass
 
+    def _get_correction(self, predicted_results: pd.DataFrame):
```

Comment:
Am I correct that we will need to update the `get_score` functions to call this function if `self.use_rank_estimation` is `True`?

Reply:
Yes